controls/KEYC-01-000005.rb

```diff
@@ -35,4 +52,25 @@
   tag stig_id: "KEYC-01-000005"
   tag cci: ["CCI-000018"]
   tag nist: ["AC-2 (4)"]
+
+  program = "/opt/keycloak/bin/kcadm.sh get events/config -r #{input('keycloak_realm')}"
+  
+  # Using inputs, testing inclusion of single element in array
+  describe json(content: command(program).stdout) do
+	  its('eventsEnabled') { should eq input('events_enabled') }
```

Comment:
Why make this an input? This has to be true to meet the requirement.

Reply:
In this case it saved needing to hard code the entire array into every control were the output was being compared to the expected output for 'eventsEnabled'. However, if it makes more sense to ensure the output includes one or a few elements from 'eventsEnabled' as you mention in another comment, an input won't be necessary.